npc/re/instances/SaraMemory.txt

```diff
@@ -1083,6 +1083,7 @@ OnDisable:
 OnEnable:
 	enablenpc instance_npcname("#MobsSara01", instance_id());
 	set .@map$, instance_mapname("1@sara");
+	killmonster .@map$, instance_npcname("#MobsSara01")+"::OnMyMobDead";
```

Comment:
Seeing the lines below this:
Would it not be better to store the event string into a variable(`.@event$`) like in faceworm as well?
This applies to all others below as well.